extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -88,7 +87,7 @@ export class WindowPortEmulator {
       if (
         e.origin == this.origin_ &&
         e.source == this.target_ &&
-        getData(e)['app'] == APP
+        e.data['app'] === APP
```

Comment:
Technically `e.data` could be null, so let's also do this: `e.data && e.data['app'] == APP`.

Reply:
Done, switched to `parseMessage` and always check if falsy before continuing.